fineract-provider/src/main/resources/db/changelog/tenant/parts/0123_transaction_summary_with_asset_owner_report_typo_fix_3.xml

```diff
@@ -0,0 +1,32 @@
ent_id JOIN m_product_loan p ON p.id = l.product_id JOIN m_external_asset_owner_transfer_details dt ON dt.asset_owner_transfer_id = t.id WHERE t.status = 'ACTIVE' and c.office_id = ${officeId} and t.settlement_date = '${endDate}' and dt.penalty_charges_outstanding_derived &gt; 0 UNION ALL SELECT '${endDate}' AS transactiondate, t.id, p.name AS product, 99999 AS transaction_type, null, '' AS chargetype, false AS reversal_indicator, 'Principal' AS Allocation_type, dt.principal_outstanding_derived AS amount, null AS asset_owner_id FROM m_external_asset_owner_transfer t JOIN m_loan l on l.id = t.loan_id join m_client c on c.id = l.client_id JOIN m_product_loan p ON p.id = l.product_id JOIN m_external_asset_owner_transfer_details dt ON dt.asset_owner_transfer_id = t.id WHERE t.status = 'BUYBACK' and c.office_id = ${officeId} and t.settlement_date = '${endDate}' and dt.principal_outstanding_derived &gt; 0 UNION ALL SELECT '${endDate}' AS transactiondate, t.id, p.name AS product, 99999 AS transaction_type, null, '' AS chargetype, false AS reversal_indicator, 'Interest' AS Allocation_type, dt.interest_outstanding_derived AS amount, null AS asset_owner_id FROM m_external_asset_owner_transfer t JOIN m_loan l on l.id = t.loan_id join m_client c on c.id = l.client_id JOIN m_product_loan p ON p.id = l.product_id JOIN m_external_asset_owner_transfer_details dt ON dt.asset_owner_transfer_id = t.id WHERE t.status = 'BUYBACK' and c.office_id = ${officeId} and t.settlement_date = '${endDate}' and dt.interest_outstanding_derived &gt; 0 UNION ALL SELECT '${endDate}' AS transactiondate, t.id, p.name AS product, 99999 AS transaction_type, null, '' AS chargetype, false AS reversal_indicator, 'Fee' AS Allocation_type, dt.fee_charges_outstanding_derived AS amount, null AS asset_owner_id FROM m_external_asset_owner_transfer t JOIN m_loan l on l.id = t.loan_id join m_client c on c.id = l.client_id JOIN m_product_loan p ON p.id = l.product_id JOIN m_external_asset_owner_transfer_details dt ON dt.asset_owner_transfer_id = t.id WHERE t.status = 'BUYBACK' and c.office_id = ${officeId} and t.settlement_date = '${endDate}' and dt.fee_charges_outstanding_derived &gt; 0 UNION ALL SELECT '${endDate}' AS transactiondate, t.id, p.name AS product, 99999 AS transaction_type, null, '' AS chargetype, false AS reversal_indicator, 'Penalty' AS Allocation_type, dt.penalty_charges_outstanding_derived AS amount, null AS asset_owner_id FROM m_external_asset_owner_transfer t JOIN m_loan l on l.id = t.loan_id join m_client c on c.id = l.client_id JOIN m_product_loan p ON p.id = l.product_id JOIN m_external_asset_owner_transfer_details dt ON dt.asset_owner_transfer_id = t.id WHERE t.status = 'BUYBACK' and c.office_id = ${officeId} and t.settlement_date = '${endDate}' and dt.penalty_charges_outstanding_derived &gt; 0)a GROUP BY a.transactiondate, a.product, a.transaction_type, a.payment_type_id, a.chargetype, a.reversal_indicator, a.Allocation_Type, a.asset_owner_id order by 1, 2, 3, 4, 5, 6, 7"/>
```

Comment:
Please share some details about the changes

Reply:
Isn't it obvious from the SQL? :laughing: 
There was a surplus space in the `' Unallocated Credit (UNC)'`. I compared it to the simple transaction summary, and this space was not there. So I removed it from the `transaction summary with asset owner` report too.